core/utils/coordinate.js

```diff
@@ -46,7 +46,7 @@ Blockly.utils.Coordinate = function(x, y) {
  * @param {?Blockly.utils.Coordinate} b A Coordinate.
  * @return {boolean} True iff the coordinates are equal, or if both are null.
  */
-Blockly.utils.Coordinate.equals = function(a, b) {
+Coordinate.equals = function(a, b) {
```

Comment:
I think these (all the ones that aren't on the prototype) should be 
`const equals = function(a, b) {...}`

based on our discussion today, right? I think so, but this is the only one of the utils classes that mixes both prototype and non-prototype methods so I'm not sure.